### DISCUSSION TITLE
workflow/deploy/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/workflow#11] [General]
# workflow/deploy/

https://shane0.github.io/workflow/deploy/

**Comment by shane0** [comment]
# deployments

**Reply by shane0** [reply]
- deployed to https://shanenull.com/cheatsheets /
- resurrect that mkdocs site + ideas for a react / supabase version
- https://github.com/shane0/photography
- root_cause/  is not in this repo yet it is for root cause analysis work in progress

**Reply by shane0** [reply]
- web3 https://b4pwb-zqaaa-aaaaf-qcc2a-cai.icp0.io/

**Reply by shane0** [reply]
https://github.com/shane0/photography

**Comment by shane0** [comment]
- try this on music/ revive the old mkdocs

**Comment by shane0** [comment]
https://shanenull.com/music/
